tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/EngSys/CodeownersTools/CodeownersTools.cs

```diff
@@ -251,13 +254,22 @@ private async Task<List<string>> CreateCodeownersPR(
             string description, // used for commit message, PR title, and PR description
             string branchPrefix,
             string identifier,
-            string workingBranch)
+            string workingBranch,
+            string repoOwner)
         {
             List<string> resultMessages = new();
             var branchName = "";
+            var hasPushPermissions = await githubService.HasPushPermission(repoOwner, repo);
+
+            if (!hasPushPermissions)
+            {
+                resultMessages.Add($"GitHub token does not have permission to push to {repoOwner} repository, opening a new PR on a branch in the main repo");
```

Comment:
not sure I understand this message. The user can't create a PR so the tool is going to create it for them?